app/javascript/spec/prov-grid/prov-vm-grid-data.js

```diff
@@ -0,0 +1,41 @@
+const headers = [['Name', 'name'],
+  ['Operating System', 'operating_system.product_name'],
+  ['Platform', 'platform'],
+  ['CPUs', 'cpu_total_cores'],
+  ['Memory', 'mem_cpu'],
+  ['Disk Size', 'allocated_disk_storage'],
+  ['Provider', 'ext_management_system.name'],
+  ['Snapshots', 'v_total_snapshots']];
```

Comment:
Do these need to be translated?

Reply:
spec file, i thought the same thing XD

Reply:
AH! I missed the spec in the directory name - thanks!